pandas/io/excel.py

```diff
@@ -824,8 +824,56 @@ class ExcelWriter(object):
 
     Notes
     -----
+    None of methods and properties are considered public.
+
     For compatibility with CSV writers, ExcelWriter serializes lists
     and dicts to strings before writing.
+
+    Examples
+    --------
+    Using ExcelWriter, some settings can be added.
+
+    Default usage.
+
+    >>> with ExcelWriter('path_to_file.xlsx') as writer:
+    ...     df.to_excel(writer)
+
+    If you want to set engine that can manipulate Excel,
```

Comment:
I think this second example is better served in `to_excel` as it generic functionality. Would be OK if you removed here and added that to the `to_excel` docs in a separate change